daos-stack/daos
# DAOS-13403 test: verify reliability and serviceability in logs

Test-tag: ControlLogEntry
Skip-unit-tests: true
Skip-fault-injection-test: true

- Add test to verify control log messages for various operations
- Add Test.random with a seed for reproducible random generation
  - Update VM tests using random to use self.random
- Add general_utils.journalctl_time() to get the time formatted
  for journalctl
  - Update tests to use this

Required-githooks: true

### Before requesting gatekeeper:


### Gatekeeper:
